src/utils.py

```diff
@@ -1014,6 +1022,52 @@ def _accumulate(distances: Collection[float]) -> float:
         return max(distances)
 
 
+class _HFFHeuristic(_RelaxationHeuristic):
+    """Implements the HFF delete relaxation heuristic.
+    """
+    @staticmethod
+    def _accumulate(distances: Collection[float]) -> float:
+        return sum(distances)
+
+    def calc_goal_h(self) -> float:
```

Comment:
is this copied from pyperplan?

Reply:
mostly yeah, a few very minor changes for style